org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/colors/XMLColorsExtensionsTest.java

```diff
@@ -71,6 +71,23 @@ public void colorOnAttr() throws BadLocationException {
 				colorInfo(1, 0, 0, 1, r(5, 14, 5, 17)));
 	}
 
+	@Test
+	public void colorOnAttrWithoutValue() throws BadLocationException {
+		// Here color is done for item/@color only
+		XMLColorsSettings settings = createXMLColorsSettings();
+		String xml = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n"
+				+ "<resources>\r\n"
+				+ "	<color name=\"opaque_red\">#f00</color>\r\n"
+				+ "	<color name=\"translucent_red\">#80ff0000</color>\r\n"
+				+ "	<color name=>BAD_COLOR</color>\r\n" // <-- here item/@color has no value

```

Comment:
Will this test the bug? The issue I had was with having an attribute that's defined as a colour that has no value (eg. `<item color =>`) but that case is not tested here

Reply:
Sorry I forget your feedback. Indeed I will fix it

Reply:
@datho7561 it should be fixed now